app/controllers/spree/api/v2/displays_controller.rb

```diff
@@ -2,11 +2,12 @@ module Spree
   module Api
     module V2
       class DisplaysController < ::Spree::Api::V2::ResourceController
+        before_action :require_spree_current_user
```

Comment:
can you please revert back require spree current user logic to the one we previously had